tests/web/test_outputs.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+from unittest.mock import MagicMock, patch
+
+pytestmark = [pytest.mark.bdb, pytest.mark.usefixtures('inputs')]
+
+UNSPENTS_ENDPOINT = '/api/v1/outputs/'
+
+
+def test_get_outputs_endpoint(client, user_pk):
+    m = MagicMock()
+    m.to_uri.side_effect = lambda s: s
+    with patch('bigchaindb.core.Bigchain.get_outputs_filtered') as gof:
```

Comment:
Would it make sense to use the patch decorator here?

Also: To my knowledge, the usual `Mock` also lets you define `side_effect`. Why `MagicMock`?